src/cpuinfo_x86.c

```diff
@@ -1047,6 +1047,24 @@ typedef struct {
   bool have_avx512;
 } OsSupport;
 
+#if defined(CPU_FEATURES_COMPILER_CLANG) || defined(CPU_FEATURES_COMPILER_GCC)
+#if defined(HAVE_CPU_INIT)
+#include <stdlib.h>
+#else
+#include "internal/filesystem.h"
+#include "internal/stack_line_reader.h"
+#include "internal/string_view.h"
+#if defined(HAVE_UTSNAME_H)
+#include <sys/utsname.h>
+#if defined(HAVE_SYSCTLBYNAME)
+#include <sys/sysctl.h>
+#endif
+#endif
```

Comment:
why defined(HAVE_SYSCTLBYNAME) is nested in #if defined(HAVE_UTSNAME_H) ?
I mean one #endif could be after <sys/utsname.h> otherwise if you always need both

Reply:
ok, just needed on Darwin but in this case, in cmake we have if(APPLE)...